neo/Ledger/NextValidatorsState.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Neo.Cryptography.ECC;
+using Neo.IO;
+using System.IO;
+
+namespace Neo.Ledger
+{
+    public class NextValidatorsState : StateBase, ICloneable<NextValidatorsState>
+    {
+        public ECPoint[] Validators = Blockchain.StandbyValidators;
+
+        public override int Size => base.Size + Validators.GetVarSize();
+
+        NextValidatorsState ICloneable<NextValidatorsState>.Clone()
+        {
+            return new NextValidatorsState
+            {
+                Validators = (ECPoint[])Validators.Clone()
```

Comment:
This is needed? maybe we can reuse the same object without clone

Reply:
This is needed. `Array` is a reference type.

Reply:
But `ECPoint` is a "readonly" class, you can't change nothing in the class i think, so we can copy the reference, isn't it?

Reply:
`ECPoint` is `readonly`. But you can modify the array. It is `ECPoint[]`.